README.md

```diff
@@ -25,29 +27,52 @@ test:
     - ./cc-test-reporter after-build --exit-code $CIRCLE_EXIT_CODE
 ```
 
-*TODO*: the above is DRAFT content.
+## Code Climate: Enterprise
+
+To report coverage to your locally-hosted *Code Climate: Enterprise* instance,
+export the `CC_TEST_REPORTER_COVERAGE_ENDPOINT` variable, or pass the
+`--coverage-endpoint` option to `after-build`.
```

Comment:
Currently, the flag is `--endpoint` or `-e`. I guess we need an issue to update this?